.github/workflows/ci.yml

```diff
@@ -91,6 +99,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           ssh-key: ${{ secrets.GH_ACTION_PRIVATE_KEY }}
+
+      - uses: extractions/netrc@v1
+        with:
+          machine: github.com
+          username: ${{ secrets.GH_ACTION_TOKEN_USER }}
+          password: ${{ secrets.GH_ACTION_TOKEN }}
+        if: vars.GOPRIVATE
```

Comment:
```suggestion
        if: ${{ vars.GOPRIVATE }}
```

Reply:
The if field didn't requires the template brackets

Reply:
OK; just for consistency the first use of the variable (line 76) uses template brackets - I'm also OK with removing them there - just to have the same in every job 🙂